.github/workflows/ci-container-image.yml

```diff
@@ -1,14 +1,17 @@
 name: Build and push latest image to container registry
 
-# This job should ideally also execute when the base image (p4lang/pi) is updated
 on:
   push:
     branches:
       - main
+  schedule:
+    # Ideally, we would update the image every time the base image (p4lang/behavioral-model)
+    # is updated, this is good enough.
+    - cron: '15 0-23/4 * * *' # "At minute 15 past every 4th hour from midnight through 23."
```

Comment:
even every midnight would be good from my pov

Reply:
this is staggered compared to the other images so that the jobs don't run at the same time and so that all images can be updated in the same "cycle"